domFoundations/selectors/script.js

```diff
@@ -0,0 +1,11 @@
+//для елементу з текстом 'Навігація по DOM дереву'
+const domNavigation = document.getElementById('headerTwo');
+
+//для першого елементу <section>
+const sectionFirstChld = document.getElementsByTagName('section')[0].firstElementChild;
+
+//для елементу списку з текстом 'Пункт 5'
+const fifthListElem = document.querySelectorAll('li')[4];
```

Comment:
Instead of using the `querySelectorAll` you need to use `querySelector` with a specific selector.